vision/Object Detection/standard_obj_text.py

```diff
@@ -80,79 +120,84 @@ def _detect_text(self, img: np.ndarray, bounds: np.ndarray) -> list:
         )
 
         ## Filter detected text to find valid characters ##
-        found_characters = []
+        found_characters: List[Tuple[str, List[Tuple[int, int]]]] = []
         for i, txt in enumerate(txt_data["text"]):
-            if (txt != None) and (len(txt) == 1):  # length of 1
+            if (txt is not None) and (len(txt) == 1):  # length of 1
                 # must be uppercase letter or number
                 if (txt.isalpha() and txt.isupper()) or txt.isnumeric():
                     # get data for each text object detected
                     x = txt_data["left"][i]
                     y = txt_data["top"][i]
-                    w = txt_data["width"][i]
-                    h = txt_data["height"][i]
+                    width = txt_data["width"][i]
+                    height = txt_data["height"][i]
 
                     # Don't continue processing if text is size of full image
                     img_h, img_w = np.shape(processed_img)
```

Comment:
Semicolons in Python? What the world has come to.

Reply:
Also should note that the semicolon fix does not work. Black will remove the semicolon and separate it like this:
```
a: int; b: int = foo()
```
will become
```
a: int
b: int = foo()
```
So, multiline is the only way this can be done.

Reply:
I know right, semicolons sound like the most sacrilegious thing to exist in a language like this. It's the only way you can write multiple statements on one line, and I agree they should be disallowed in nearly all circumstances. Unfortunately, type hinting tuple unpacking can get crazy and add a lot of lines. 
You almost had it correct. Tuple unpacking type annotations always have to be done immediately before the unpacking, so the right way to do it would've been:
```
a: int; b: int
a, b = foo()
```
I thought that black would allow semicolons in this case, but that's my bad. Black actually completely doesn't allow them (understandably) for the reason that they are not pythonic. There's also no way for me to add an exception for semicolons. So, you're gonna have to do it the way [PEP 526](https://www.python.org/dev/peps/pep-0526/#global-and-local-variable-annotations) recommends it, which unfortunately uses an extra line for each element of the tuple
```
a: int
b: int
a, b = foo()
```